app/customer.py

```diff
@@ -0,0 +1,46 @@
+import dataclasses
+import datetime
+from typing import List, Any
+
+from app.car import Car
+
+
+@dataclasses.dataclass
+class Customer:
+    name: str
+    product_cart: dict
+    car: Car
+    money: int
+
+    @classmethod
```

Comment:
Why you use `classmethod` here?

Reply:
If you use classmethod, you should use cls instead of Customer() in the code of this method. It might make sense to use it, if you would have child classes that reuse this method. Does this remark make sense to you now?